ARCs/arc-0034.md

```diff
@@ -0,0 +1,132 @@
+- A thorough understanding of the needs of the community
+- Alignment of personal interests with the advancement of the Algorand ecosystem.
+> Issues requiring resolution may have been discussed on various online platforms such as forums, discord, and social media networks.
+
+### Voting System
+The voting system for the xGov program will be the following:
+- Only members of xGov are eligible to participate in the voting process
+- Utilization of the Algo committed by xGov as a form of voting currency, with a 1:1 ratio of Algos to votes
+- Proposals that do not meet the minimum threshold of votes required will be removed from consideration.
+
+### Threshold 
+In order for a proposal to be approved, it is necessary for the number of votes in favor of the proposal to be proportionate to the amount of funds requested. This ensures that the allocation of funds is in line with the community's consensus and in accordance with democratic principles.
+
+> eg. 2 000 000 Algo are available to be given away as grants, 200 000 000 millions Algo are committed to the xGov Process (200 000 000 votes available):
+> - Proposal A request 100 000 Algos (5 % of the Amount available)
+> - Proposal A needs 5% of the votes (10 000 000 Votes) to go through
+
+### Life of a proposal
+The proposal process will have the same duration as the governance period. Within the 3 months, the following steps will be implemented:
+- Anyone can submit a proposal at any time. Proposals will be hidden to prevent bias.
+- 0.5 months after the start of a governance periode, proposals submitted will be revealed. 
+- 1 month is allocated for proposals refinement and temperature check. Proposals will be evaluated and refined by the community and xGovs.
+- 1 month is allocated for voting on proposals. The community will vote on proposals that have passed the refinement and temperature check stage.
+- 0.5 months for negotiation and claiming grants. Proposals that have been approved will be negotiated and grants will be claimed.
+
+### Submit a proposal
+In order to submit a proposals, a proposer needs to fill a form on the website provided for this purpose: <a href="">URL TBD</a>.
+
+#### Information requried to submit a proposal
+``` json 
+{
+   "Title": "title of the proposal",
+   "ProposerEmail": "",
+   "Country": "",
+   "CompanyName":"",
+   "Team": [],
+   "ExperienceOnAlgorand": {"Low", "Medium", "High"}, 
+   "Category": {
+    "dApps", 
+    "Tools", 
+    "Community"
+    },
+   "Area": {
+    "Banking", 
+    "Defi", 
+    "DEX", 
+    "Gaming", 
+    "Identity", 
+    "Marketplace", 
+    "Metaverse", 
+    "NFT", 
+    "Oracle", 
+    "Storage",
+    "User Onboarding",
+    "IDE",
+    "Teal",
+    "Deployment",
+    "Libraries",
+    "Monitoring",
+    "Node",
+    "Education",
+    "Other"
+    },
+   "OpenSource": {true, false},
+   "AmountRequested": 0,
+   "RoapMap": "Explanation of what will be achieved",
```

Comment:
```suggestion
   "Roadmap": "Explanation of what will be achieved",
```
@SudoWeezy I wasn't sure if this was a typo but I just wanted to doublecheck with you